### ISSUE TITLE
Unable to allocate 4.00 GiB for an array with shape (134217728, 4) and data type float64

### ISSUE THREAD

**Issue by Seruios** [facebookresearch/pifuhd#193]
AND

File "S:\Python\pifuhd-main\lib\data\EvalWPoseDataset.py", line 125, in fullbody_crop
    center = (0.5 * (pmax[:2] + pmin[:2])).astype(np.int)
  File "S:\Python\lib\site-packages\numpy\__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'inf'?

**Comment by lukas735** [comment]
Yes its true. Its deprecated... But if all know how easy it is to fix. Why not changing the script and make a Commit. It is not really good practice to provide a outdated script which not anymore work. I also got this error and several more. All has to do with anything outdated... or maybe its also my Windows... lol. Don't know because i still have no fix for the more problematic GlutInit error.... 

So for all who has this problem go into all Scripts mentioned in your Traceback and search for `np.int` and `np.bool`
np.int and np.bool is deprecated since numpy v1.2 remove the np. and use int and bool instead.
